package.json

```diff
@@ -19,17 +19,21 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "lint": "aegir-lint",
-    "coverage": "gulp coverage",
-    "test": "gulp test --dom",
-    "test:node": "npm run test:unit:node",
-    "test:browser": "npm run test:unit:browser",
-    "test:unit:node": "gulp test:node",
-    "test:unit:node:core": "TEST=core npm run test:unit:node",
-    "test:unit:node:http": "TEST=http npm run test:unit:node",
-    "test:unit:node:gateway": "TEST=gateway npm run test:unit:node",
-    "test:unit:node:cli": "TEST=cli npm run test:unit:node",
-    "test:unit:browser": "gulp test:browser",
+    "lint": "aegir lint",
+    "build": "aegir build",
+    "test": "aegir test --target node --target browser --no-cors",
+    "test:node": "aegir test --target node",
+    "test:browser": "aegir test --target browser --target webworker --no-cors",
+    "release": "aegir release --target node --target browser",
+    "release-minor": "aegir release --type minor --target node --target browser",
+    "release-major": "aegir release --type major --target node --target browser",
+    "coverage": "aegir coverage",
+    "test:unit:node": "aegir test --target node",
+    "test:unit:node:core": "aegir test --target node -f test/core/*.js",
+    "test:unit:node:http": "aegir test --target node -f test/http-api/index.js",
+    "test:unit:node:gateway": "aegir test --target node -f test/gateway/index.js",
+    "test:unit:node:cli": "aegir test --target node -f test/cli/index.js",
+    "test:unit:browser": "aegir test --target browser --no-cors",
     "test:interop": "npm run test:interop:node",
     "test:interop:node": "mocha -t 60000 test/interop/node.js",
     "test:interop:browser": "mocha -t 60000 test/interop/browser.js",
```

Comment:
This needs to start using aegir due to https://github.com/ipfs/js-ipfs/pull/830/files#r149579164

Reply:
I'll get those in.